srcpkgs/compfy/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'compfy'
+pkgname=compfy
+version=1.7.1
+revision=1
+build_style=meson
+configure_args="-Dupdate_checks=true"
```

Comment:
Typically we disable update checking in packages.

Reply:
Removed the check updates.

Reply:
anything else needed??